ini_multiline_value_fix_test.go

```diff
@@ -0,0 +1,60 @@
+package ini_test
+
+import (
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+	"path/filepath"
+	"testing"
```

Comment:
Please refer to other source files to spilt these into two groups.

Reply:
OK, done